introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-incremental-training-highlevel.ipynb

```diff
@@ -10,9 +10,15 @@
     "  1. [Permissions and environment variables](#Permissions-and-environment-variables)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    deploy_amt_model = True
</pre><p>Same here, why you do not have a separate section with explanation for deploy_amt_model&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3658/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
The paragraph explaining its usage is a bit above (and it's in all notebooks). In some notebooks the flag is on its own, in others it's part of the main code block. I started putting it in its own code block some point, and didn't update the previous notebooks cause I thought it wasn't necessary but let me do that, it's cleaner.